dist/ramda.js

```diff
@@ -2330,6 +2335,28 @@
     });
 
     /**
+     * Returns `true` if no elements of the list match the predicate,
+     * `false` if there are any that do.
+     *
+     * @func
+     * @memberOf R
+     * @category List
+     * @sig (a -> Boolean) -> [a] -> Boolean
+     * @param {Function} fn The predicate function.
+     * @param {Array} list The array to consider.
+     * @return {Boolean} `true` if the predicate is not satisfied by every element, `false` otherwise.
+     * @example
+     *
+     *      var xs = R.map(parseInt, ['1', '2', '3', 'foo']);
+     *      xs; //=> [1, 2, 3, NaN]
+     *      R.none(isNaN)(xs); //=> false
+     *      xs.pop();
```

Comment:
should we merge this, i don't think we should have the example *mutate* the list. Better to create e.g. `ys = [1, 2, 3]`